pkgs/development/tools/jq/default.nix

```diff
@@ -29,17 +29,20 @@ stdenv.mkDerivation rec {
 
   configureFlags =
     [
-    "--bindir=\${bin}/bin"
-    "--sbindir=\${bin}/bin"
+    "--bindir=\${out}/bin"
+    "--sbindir=\${out}/bin"
     "--datadir=\${doc}/share"
     "--mandir=\${man}/share/man"
     ]
-  # jq is linked to libjq:
+    # jq is linked to libjq:
     ++ stdenv.lib.optional (!stdenv.isDarwin) "LDFLAGS=-Wl,-rpath,\\\${libdir}";
 
-  installCheckPhase = "$bin/bin/jq --help >/dev/null";
   doInstallCheck = true;
-  doCheck = true;
```

Comment:
I’d rather enable these tests and disable them selectively on darwin.

Reply:
I don't know what you mean by that, this doesn't disable any tests.

Reply:
There is `postInstallCheck`.

Reply:
> I don't know what you mean by that, this doesn't disable any tests.

You quite obviously removed `doCheck = true`.

Why does it fail with `doCheck` (which runs `make check`), but succeed with `installCheckTarget = "check"` then?

Reply:
All this does is run the tests after the libraries are installed installed to $lib instead of before, the tests failed because the libjq wasn't in the expected install location yet.

Reply:
I wonder why it worked on linux then.

Reply:
The way libraries are linked is a bit different on darwin, we link directly against libraries using their absolute path instead of using RPATH. I could have worked around it with DYLD_LIBRARY_PATH, but that's a bit harder and probably more brittle.